tools/wpt/testfiles.py

```diff
@@ -237,6 +242,9 @@ def affected_testfiles(files_changed, skip_tests, manifest_path=None):
             full_path = os.path.join(wpt_root, repo_path[1:].replace("/", os.path.sep))
         nontest_changed_paths.add((full_path, repo_path))
 
+    interface_name = lambda x: re.match('.*/([^/]*)\.idl$', x).group(1)
```

Comment:
`os.path.splitext(os.path.basename(x))[0]` would be clearer here I think. Possibly not exactly the same for every input, but I can't think of one where it would matter.

Reply:
Testing the new hotness:

```suggestion
    interface_name = os.path.splitext(os.path.basename(x))[0]
```

Reply:
Neeeeeat! You should use that next time @foolip (hit the little `add suggestion` button)

Reply:
What, what's this new feature?! I will use it!

Reply:
Hmm, I don't see it. Maybe it's an A/B test? Where can I find it?